src/Router/index.js

```diff
@@ -148,10 +165,14 @@ class Router {
      * @param response
      * @return {*}
      */
-    static dispatchRoute(route, response) {
+    static async dispatchRoute(route, response) {
```

Comment:
It doesn't have to be async, it should also allow the response that is NOT promise.

Reply:
async/await construction CAN work with not a promise response.

Reply:
It does, but it always returns a Promise, doesn't it? I don't see promise handler added on `respondToRoute` function, have you tested it?

Reply:
`handlerResponse` is getting the response with `await`, so `respondToRoute` can be as is

Reply:
And, of course I have tested all the fixes.